src/main/msp/msp.c

```diff
@@ -2074,6 +2075,31 @@ case MSP_NAME:
 #endif
         break;
 
+    case MSP2_SENSOR_CONFIG_ACTIVE:
+        // use sensorIndex_e index: 0:GyroHardware, 1:AccHardware, 2:BaroHardware, 3:MagHardware, 4:RangefinderHardware
+#if defined(USE_ACC)
+        sbufWriteU8(dst, detectedSensors[SENSOR_INDEX_ACC]);
+#else
+        sbufWriteU8(dst, ACC_NONE);
+#endif
+#ifdef USE_BARO
+        sbufWriteU8(dst, detectedSensors[SENSOR_INDEX_BARO]);
+#else
+        sbufWriteU8(dst, BARO_NONE);
```

Comment:
I'd send special 'SENSOR_NOT_AVAILABLE' value in this case - this is new code and it may be useful if configurator knows that sensor support is not enabled in firmware